C7/C7.ini

```diff
@@ -0,0 +1,2 @@
+[audio]
+musicVolume = 100
```

Comment:
Minor: We shouldn't commit this file as doing so will cause our settings to get bundled as the default (this is mostly on me as I didn't remember to add it to .gitignore when it was added).

Reply:
I usually try not to commit extraneous things but this one must have slipped through. I'll delete it.